src/Command/Extension/SetExtensionVersionCommand.php

```diff
@@ -58,6 +60,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
+        $dryRun = ($input->getOption('dry-run') !== false);
```

Comment:
I think we can just typecast `$input->getOption('dry-run')`. It can only be `TRUE` or `FALSE`.

Reply:
According the return type iirc it's string|bool|null, that's why would not do a type cast here but a strict check. But the condition should ne === true. Will fix...